gst/renderer/gstttmlrender.c

```diff
@@ -343,8 +343,20 @@ gst_ttml_render_negotiate (GstTtmlRender * render, GstCaps * caps)
 
   original_caps = caps;
 
+  GstStructure const* const structure = gst_caps_get_structure (caps, 0);
+  gint overlay_composition_width = 0, overlay_composition_height = 0;
+  if (gst_structure_get_int (structure, "overlay-composition-width", &overlay_composition_width) &&
+      gst_structure_get_int (structure, "overlay-composition-height", &overlay_composition_height)) {
```

Comment:
When downstream connection (`kmssink` in our case) sets these two propreties, and the renderer is capable to attach overlay composition rather than blend into video, the overlay composition resolution is fixed.

We send the caps back downstream as well, and the renderer will be able to recognize fixed resolution.

These properties are introduced by us.

The idea is that physical monitor is known and fixed in our AERQ case, and in general it does not make a lot of sense to downsize the subtitles if the video is low res. But also, it does not make a lot of sense to render 4K and upload to slow buffer if this is sensed as performance hit.